core/src/test/scala/org/http4s/rho/CompileRoutesSuite.scala

```diff
@@ -40,16 +39,16 @@ class CompileRoutesSuite extends FunSuite {
         (PUT / "hello" |>> "PutFoo") :: Nil
 
     val srvc = CompileRoutes.foldRoutes[IO](routes)
-    assertEquals("GetFoo", RRunner(srvc).checkOk(Request(uri = uri"/hello")))
-    assertEquals("PutFoo", RRunner(srvc).checkOk(Request(method = Method.PUT, uri = uri"/hello")))
+    assertIO(RRunner(srvc).checkOk(Request(uri = uri"/hello")), "GetFoo")
+    assertIO(RRunner(srvc).checkOk(Request(method = Method.PUT, uri = uri"/hello")), "PutFoo")
   }
 
   test("A CompileService should concatenate correctly") {
     val c1 = RoutesBuilder[IO](); getFoo(c1)
     val c2 = RoutesBuilder[IO](); putFoo(c2)
 
     val srvc = c1.append(c2.routes()).toRoutes()
-    assertEquals("GetFoo", RRunner(srvc).checkOk(Request(uri = uri"/hello")))
-    assertEquals("PutFoo", RRunner(srvc).checkOk(Request(method = Method.PUT, uri = uri"/hello")))
+    assertIO(RRunner(srvc).checkOk(Request(uri = uri"/hello")), "GetFoo")
+    assertIO(RRunner(srvc).checkOk(Request(method = Method.PUT, uri = uri"/hello")), "PutFoo")
```

Comment:
```suggestion
    assertIO(RRunner(srvc).checkOk(Request(uri = uri"/hello")), "GetFoo") *>
    assertIO(RRunner(srvc).checkOk(Request(method = Method.PUT, uri = uri"/hello")), "PutFoo")
```

or 

```suggestion
for {
    _ <- assertIO(RRunner(srvc).checkOk(Request(uri = uri"/hello")), "GetFoo")
    _ <- assertIO(RRunner(srvc).checkOk(Request(method = Method.PUT, uri = uri"/hello")), "PutFoo")
} yield ()
```

There are quite a few like this in this file.

Reply:
😮 shame on me. I've fixed that in all files.